Makefile

```
@@ -104,6 +104,21 @@ markdownlint: ## Lint Markdown files
 nixie: ## Validate Mermaid diagrams
 	$(NIXIE) --no-sandbox
 
+audit: rust-audit ## Audit dependencies for known vulnerabilities
+
+rust-audit: ## Audit every Rust manifest for known vulnerabilities
```

checkmake (0.3.2)

[warning] Target body for "rust-audit" exceeds allowed length of 5 lines (11).

(maxbodylength)